packages/db/src/models/async_operation.ts

```diff
@@ -0,0 +1,15 @@
+import { BasePgModel } from './base';
+import { tableNames } from '../tables';
+
+import { PostgresAsyncOperation, PostgresAsyncOperationRecord } from '../types/async_operation';
+
+// eslint-disable-next-line max-len
+class AsyncOperationPgModel extends BasePgModel<PostgresAsyncOperation, PostgresAsyncOperationRecord> {
+  constructor() {
+    super({
+      tableName: tableNames.asyncOperations,
+    });
+  }
+}
+
+export { AsyncOperationPgModel };
```

Comment:
This model is actually added in another PR as well:

https://github.com/nasa/cumulus/pull/2033/files#diff-9ea14a7792723723e953031b1d1b7bbc4341c2c8ef86d45ccd7af2e25dba62e2

Adding it here so I can use the base model's `getRecordCumulusId` for async operations.